### ISSUE TITLE
Godot won't run after updating to version 3.0.3-1

### ISSUE THREAD

**Issue by mrwanashraf** [godotengine/godot#19721]
**Godot version:**
3.0.3-1

**OS/device including version:**
OS: Manjaro 17.1.10 Hakoila
Device: x86_64 Linux 4.14.48-2-MANJARO
GPU: GeForce 920M
DE: KDE 5.46.0 / Plasma 5.12.5

**Issue description:**
an error shows whenever i try to run godot after updating it to version 3.0.3-1, i tried to run godot with the default method, also with primusrun & optirun but it still showed the same error.

the error message is:
```
ERROR: initialize: Condition ' !fbc ' is true. returned: ERR_UNCONFIGURED
   At: platform/x11/context_gl_x11.cpp:121.
ERROR: initialize: Condition ' x11_window == 0 ' is true. returned: ERR_UNAVAILABLE
   At: platform/x11/os_x11.cpp:345.
```

**Steps to reproduce:**
simply running godot from panel, or running `godot` , `primusrun godot` or `optirun godot` from terminal would produce the same error.

**Comment by akien-mga** [comment]
Are you using Plasma on Wayland? If so, do you have xwayland installed?

**Comment by mrwanashraf** [comment]
@akien-mga  no I'm running plasma on X11

**Comment by mrwanashraf** [comment]
i tried to run godot today and it ran without issues, weird.. anyways since the problem is not there anymore i will have to close the issue.

**Comment by TheShed412** [comment]
Recently I decided to try and run Godot on a Linux VM in Virtualbox and I was getting this same error on start up. When I disabled 3D acceleration that fixed it. While mine has to do with VB being weird with 3D acceleration, this issue does stem from 3D Acceleration going wrong somehow.

Don't really know if it's addressable from our end! Just thought I'd narrow it down if this crops up in the future.

VirtualBox Version: 
5.2.16

Host OS:
Windows 10 Home 
Version 1803

Guest:
Linux Mint 18.3
Cinnamon DE
